etl/steps/data/garden/homicide/2024-10-30/unodc.meta.yml

```diff
@@ -0,0 +1,88 @@
+# NOTE: To learn more about the fields, hover over their names.
+definitions:
+  common:
+    presentation:
+      topic_tags:
+        - Homicides
+
+  metric:
+    <% if unit_of_measurement == "Counts" %>
+    Number of homicides
+    <% elif unit_of_measurement == "Rate per 100,000 population" %>
+    Homicide rate per 100,000 population
+    <%- endif -%>
+  sex:
+    <% if sex == "Total" %> all victimes<% elif sex == "Male" %>male victims<% elif sex == "Female" %>female victims<%- endif -%>
+  age:
+    <% if age == "Total" %>
+    in all age-groups
+    <% elif age == "30-44" %>
+    aged 30-44 years
+    <% elif age == "45-59" %>
+    aged 45-59 years
+    <% elif age == "60 and older" %>
+    aged over 60 years
+    <% elif age == "0-9" %>
+    0-9 years
+    <% elif age == "10 -14" %>
+    10-14 years
+    <% elif age == "15 -17" %>
+    15-17 years
+    <% elif age == "18-19" %>
+    18-19 years
+    <% elif age == "20-24" %>
+    20-24 years
+    <% elif age == "25-29" %>
+    25-29 years
+    <% elif age == "Unknown" %>
+    of unknown age
+    <%- endif -%>
+  unit:
+    <% if unit_of_measurement == "Counts" %>
+    homicides
+    <% elif sex == "Rate per 100,000 population" %>
+    homicides per 100,000 population
+    <%- endif -%>
+# Learn more about the available fields:
+# http://docs.owid.io/projects/etl/architecture/metadata/reference/
+dataset:
+  update_period_days: 365
+
+
+tables:
+    by_mechanisms:
+      variables:
+        value:
+          title: |
+            {definitions.metric} - << category >>
+          description_short: |
+           {definitions.metric} of {definitions.sex} {definitions.age} where the weapon was << category.lower() >>
+          unit: |
+            {definitions.unit}
+    by_relationship_to_perpetrator:
+      variables:
+        value:
+          title: |
+            {definitions.metric} - << category >> - sex: << sex >>
```

Comment:
Is there a reason you are not using {definitions.sex} and {definitions.age} in the title as well? Especially for age I think it would make the titles much nicer.

Reply:
Good point, I was being a little lazy in trying to match to the existing variable names in the previous dataset. I can make a nicer version for presentation.title_public.

Reply:
Yeah, that makes sense! Thank you! :)